packages/Webkul/Installer/src/Console/Commands/DefaultUser.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+namespace Webkul\Installer\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+
+use function Laravel\Prompts\select;
+use function Laravel\Prompts\text;
+
+class DefaultUser extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'unopim:user:create
+        {--name= : The name of the user}
+        {--email= : The email address of the user}
+        {--password= : The password for the user}
+        {--ui_locale= : The UI locale (e.g., en_US) of the user}
+        {--timezone= : The timezone of the user}
+        {--admin : Specify if the user is an admin}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This command allows you to create a new user with a name, email, password, UI locale, timezone, and optionally specify whether the user is an admin.';
+
+    /**
+     * Locales list.
+     *
+     * @var array
+     */
+    protected $locales = [
+        'ar_AE'       => 'Arabic',
+        'de_DE'       => 'German',
+        'en_US'       => 'English',
+        'es_ES'       => 'Spanish',
+        'fr_FR'       => 'French',
+        'hi_IN'       => 'Hindi',
+        'ja_JP'       => 'Japanese',
+        'nl_NL'       => 'Dutch',
+        'ru_RU'       => 'Russian',
+        'zh_CN'       => 'Chinese',
+    ];
+
+    /**
+     * Create UnoPim default user.
+     */
+    public function handle()
+    {
+
+        $this->loadEnvConfigAtRuntime();
+
+        $adminName = $this->option('name') ?: text(
+            label: 'Set the Name for Administrator',
+            default: 'Admin',
+            required: true
+        );
+
+        $adminEmail = $this->option('email');
+
+        if (! $adminEmail || ! filter_var($adminEmail, FILTER_VALIDATE_EMAIL)) {
+            $adminEmail = text(
+                label: 'Provide Email of Administrator',
```

Comment:
This is a command to create a new user, that user if has all permissions will be an Administrator, Use the word User here instead of 'Administrator'
```php
   abel: 'Provide Email of Administrator',

```